### PR TITLE
Standardize pit-loss baseline to drive-through and add fixed 2.75s pit-box transition allowance

### PR DESCRIPTION
**Pull request by Lalabot77**
### Motivation
- Recent test data showed a consistent ~2.75s overhead between a clean drive-through and a stop-and-go in the pit lane, which should be modeled as a fixed boxed-transition allowance rather than learned into the pit-lane baseline.  
- Making the learned/stored pit-lane loss explicitly a drive-through-only baseline prevents ambiguity and avoids accidental double-counting when a stopped-box prediction adds transition overhead.  
- The change keeps pit-loss learning ownership with the Pit Timing subsystem while applying the fixed allowance only where stopped-box prediction needs it (not to pure lane-travel outputs).  

### Description
- Added a new constant `PitExitTransitionAllowanceSec = 2.75` and applied it in the shared prediction seam by adding the value to `CalculateTotalStopLossSeconds()` so boxed-stop consumers receive `pit-lane baseline + boxed service + 2.75s`.  
- Updated canonical docs and user guidance to require drive-through-only baseline learning and to document the new boxed-stop contract (`boxed stop loss = learned drive-through baseline + boxed service model + 2.75s transition allowance`).  
- Files changed: `LalaLaunch.cs` (constant + total-stop-loss composition), `Docs/Subsystems/Pit_Timing_And_PitLoss.md`, `Docs/Internal/SimHubParameterInventory.md`, `Docs/Quick_Start.md`, `Docs/User_Guide.md`, `Docs/Internal/Development_Changelog.md`, and `Docs/RepoStatus.md`.  
- Scope/ownership preserved: pure lane-travel outputs (`Fuel.LastPitLaneTravelTime`, `PitExit.TimeS`) were left unchanged and Opponents remains the race-scoped pit-exit prediction owner; no new log lines were added.  

### Testing
- Ran repository searches (`rg`) to confirm the new symbol and updated doc text (`PitExitTransitionAllowanceSec`, `CalculateTotalStopLossSeconds`, and references to drive-through baseline`), which returned the expected matches.  
- Attempted a project build (`dotnet build`) in the environment but the `dotnet` SDK is not installed here, so a compile check could not be executed in this CI-like sandbox.  
- No automated unit tests were present or executed as part of this change; behavior verification is documented and should be validated in an environment with the SDK and live telemetry (recommended: run smoke build + a short live pit sequence to confirm UI/exports and that pure lane-travel outputs are unchanged).

------
[Codex Task](https://chatgpt.com/codex/cloud/tasks/task_e_69dd741b9444832f8eec11a235b219f6)

### CODE REVIEW

File: LalaLaunch.cs [hunk 1]
```diff
@@ -13028,8 +13029,7 @@ private double CalculateTotalStopLossSeconds()
             double modeledBoxTargetSec = CalculatePitBoxModeledTargetSeconds();
             double repairRemainingSec = CalculatePitBoxRepairRemainingSeconds();
             double boxTime = Math.Max(modeledBoxTargetSec, repairRemainingSec);
-
-            double total = pitLaneLoss + boxTime;
+            double total = pitLaneLoss + boxTime + PitExitTransitionAllowanceSec;
```
[the remark this answers is not in the document]
Comment (Lalabot77) [reply LalaLaunch.cs:13032]: @codex - Good catch. Please fix PR 566 so `PitExitTransitionAllowanceSec = 2.75` is applied only for boxed-stop prediction, not unconditionally in `CalculateTotalStopLossSeconds()`.

Problem:
`CalculateTotalStopLossSeconds()` currently adds the 2.75s allowance for all pit trips. That method feeds `pitLossSec` into the pit-exit predictor, so drive-through / no-box pit-lane cases are now overestimated.

Required fix:
- apply the fixed 2.75s transition allowance only when a boxed stop is actually expected/relevant
- do not apply it for pure drive-through / no-box pit lane travel
- keep `Fuel.Live.PitLaneLoss_S` as drive-through baseline
- keep pure lane-travel outputs unchanged
- update docs if the final gating semantics need clarification

Important:
- do not use a naive gate that only works once already in the box if that makes early stop prediction useless
- instead, find the smallest clean existing seam that distinguishes boxed-stop intent/expectation from pure drive-through behavior
- keep scope tight
